pipeline/outputs/datadog.md

```diff
@@ -24,7 +24,8 @@ Before you begin, you need a [Datadog account](https://app.datadoghq.com/signup)
 | dd_source       | _Recommended_ - A human readable name for the underlying technology of your service (e.g. `postgres` or `nginx`). If unset, Datadog will look for the source in the [`ddsource` attribute](https://docs.datadoghq.com/logs/log_configuration/pipelines/?tab=source#source-attribute).                                                                                                                                                                                                                                                                                                                   |                                  |
 | dd_tags         | _Optional_ - The [tags](https://docs.datadoghq.com/tagging/) you want to assign to your logs in Datadog. If unset, Datadog will look for the tags in the [`ddtags' attribute](https://docs.datadoghq.com/api/latest/logs/#send-logs).                                                                                                                                                                                                                                                                                                                                   |                                  |
-| dd_message_key  | By default, the plugin searches for the key 'log' and remap the value to the key 'message'. If the property is set, the plugin will search the property name key.                                                                                                                                                                                                                                                                          |                                  |
+| dd_message_key  | By default, the plugin searches for the key 'log' and remap the value to the key 'message'. If the property is set, the plugin will search the property name key.                                                                                                                                                                                                                                                                                                                                                                                                       |                                  |
+| dd_hostname     | The host that emitted logs should be associated with. If unset, Datadog will expect the host to be set as `host`, `hostname`, or `syslog.hostname` attributes. See Datadog Logs preprocessor documentation for up-to-date recognized attributes.                                                                                                                                                                                                                                                                                                                       |                                  |
```

Comment:
The previously mentioned question is still applicable, but this suggestion updates the content to fit the new style guide, including having values be listed in ticks and not single quotes, and to not have empty cells in tables. It also makes a few wording changes for clarity (hopefully the content is still correct based on those).

```suggestion
| dd_message_key  | By default, the plugin searches for the key `log` and remaps the value to the key `message`. If the property is set, the plugin searches the property name key. | _none_ |
| dd_hostname     | The host the emitted logs should be associated with. If unset, Datadog expects the host to be set with `host`, `hostname`, or `syslog.hostname` attributes. See Datadog Logs preprocessor documentation for recognized attributes. | _none_ |
```

Reply:
Thanks @cnorris-cs ! The suggestion looks good to me. I applied it.